build/template-publish-packages-and-symbols.yaml

```diff
@@ -25,11 +25,11 @@ steps:
     publishVstsFeed: '46419298-b96c-437f-bd4c-12c8df7f868d'
     allowPackageConflicts: true
   continueOnError: true
-  condition: eq(${{ parameters.PublishArtifacts }}, 'true')
+  condition: eq(variables['PublishNugetAndSymbols'], 'true')
 
 - task: PublishSymbols@2
   displayName: 'Publish symbols'
   inputs:
     SearchPattern: '${{ parameters.SymbolPublishWildcard }}'
     SymbolServerType: TeamServices
-  condition: eq(${{ parameters.PublishArtifacts }}, 'true')
+  condition: eq(variables['PublishNugetAndSymbols'], 'true')
```

Comment:
Does this variable need to be passed in from the top level yaml to work, like here: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/b529f3024f4aed17aa9bdc48e23294f03a5d18e6/build/pipeline-releasebuild.yaml#L6-L10

Reply:
Yes this will be passed as a build variable. I split them into `PublishToVSTSFeed` and `PublishSymbols`  variables. 

![image](https://user-images.githubusercontent.com/90415114/191830504-b58e1fb4-0dd4-4554-a797-937690f502e0.png)

Ran a [test build](https://identitydivision.visualstudio.com/IDDP/_build/results?buildId=995169&view=logs&j=9906ff93-67e4-5c55-442d-8f18d4fd3348&t=daaef45a-0156-5701-28d1-cab03afd58b7), without the variable and confirmed that these are not executed 

![image](https://user-images.githubusercontent.com/90415114/191830774-328a0ede-9a0c-49d1-ac00-91790c8074f1.png)

and[ ran one with the variables ](https://identitydivision.visualstudio.com/IDDP/_build/results?buildId=995192&view=logs&j=9906ff93-67e4-5c55-442d-8f18d4fd3348)and these were pushed to the feed, 

![image](https://user-images.githubusercontent.com/90415114/191830890-f83dc40a-0ce0-4d42-aca9-527b35881546.png)

Basically with this fix, the default is turned off so we are not publishing it nightly or for test builds. Even for release build we should not set this, instead we should rely on the publish package pipeline. This variable should be set only when we know there is an internal release that needs to be pushed.

Reply:
@pmaytak